Sources/Runtime/generics.cpp

```diff
@@ -4,7 +4,7 @@
 
 using namespace trill;
 
-void *trill_createGenericBox(void *typeMetadata, void *witnessTable) {
+void *trill_createGenericBox(const void *typeMetadata, const void **witnessTable) {
```

Comment:
I realized it is actually a `void **`. It's a pointer to function pointers. It doesn't need a count, though, because accesses into this table will be done at compile time.